src/components/Sections/SectionTable.less

```diff
@@ -2,18 +2,15 @@
   overflow-x: hidden;
   margin-top: 0 !important;
 
-  thead {
-    th {
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
-  }
-
   th, td {
+    padding: 3px !important;
     border-top: 1px solid #DADBDB !important;
     border-left: 1px solid #DADBDB !important;
     border-bottom: none !important;
     border-right: none !important;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    width: 80px;
```

Comment:
why we added a fixed width ?

Reply:
This acts like a minimum width (if there are a lot of items it will display only the first ones that fits)